example/app.js

```diff
@@ -23,82 +23,82 @@ function addEventBlock(text, headerText, alertLevel) {
 function updateIdFields() {
   var userIdSpan = document.getElementById('userId');
   var tenantIdSpan = document.getElementById('tenantId');
-  userId.textContent = Centricient.getUserId();
-  tenantId.textContent = Centricient.getTenantId();
+  userId.textContent = Quiq.getUserId();
+  tenantId.textContent = Quiq.getTenantId();
```

Comment:
The new SDK version exposes a global `Quiq` object instead of a `Centricient` object. The contents and behavior should be exactly the same. You'll just need to change the variable names throughout.